tests/test_markdown.py

```diff
@@ -1,16 +1,19 @@
 import pathlib
 import pytest
+from shutil import which
+from unittest.mock import Mock
 
-from mktestdocs import check_md_file
+from mktestdocs import check_md_file, register_executor
+from mktestdocs.__main__ import exec_bash
 
 
 @pytest.mark.parametrize("fpath", pathlib.Path("tests/data/good").glob("*.md"), ids=str)
 def test_files_good(fpath):
     check_md_file(fpath=fpath)
 
 
-@pytest.mark.parametrize("fpath", pathlib.Path("tests/data/bad").glob("*.md"), ids=str)
-def test_files_bad(fpath):
+def test_files_bad():
```

Comment:
Why aren't you also testing `b.md` here?

Reply:
`b.md` has no python blocks, only bash, so no exceptions will be raised and the test would fail. Previously this glob was a list of just one file, `a.md` so it's testing the same data.

Reply:
Ah right! Good point.